src/vtbackend/Terminal.cpp

```diff
@@ -1040,7 +1040,7 @@ void Terminal::updateHoveringHyperlinkState()
     auto const oldState = _hoveringHyperlinkId.exchange(newState);
 
     if (newState != oldState)
-        screenUpdated();
+        renderBufferUpdated();
```

Comment:
I am just updating render buffer here, is there any need to flush input on hovering state update event?

Reply:
No, there isn't. I am however not sure if there are any other implications to this. I do not think so right out of my head, however. 🤔